VALUES-CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,23 @@
+Our code of conduct are the measures which ensure that we stick to our values
+at every stage of our work and all over at our organisation.
+
+
+## Values
+- ScaLabs is both labs & people
+- Functioning societies produce wonder
```

Comment:
- We as an organization are super-national
- Catching possibilities beyond administrative borders